public/scss/utilities/_layout.scss

```diff
@@ -1,3 +1,7 @@
+// AD-HOC utility classes for composing simple layouts without defining new classes
+// intentionally structured after Tailwind classes
```

Comment:
I debated about adding these, but it felt like there were so many small examples of just needing a bit of spacing here and there that allows avoiding creating a lot of selectors for repeat properties. They're modeled after Tailwind to try and remove ambiguity. Hopefully it's not too confusing to mix the few and will increase reusability.